### PR TITLE
[ZEPPELIN-751]Log an error and continue reading the remaining notebooks

### PR DESCRIPTION
**Pull request by mwkang**
### What is this PR for?
Log an Amazon Service Exception.
Continue reading the remaining notebooks into S3ObjectSummary.

### What type of PR is it?
Bug Fix

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-751

### Questions:
* Do you have a plan using Guava?
(Sometimes Optional is good stuff)

### CODE REVIEW

**Comment by Leemoonsoo** [comment]
Changes looks good to me!

**Comment by mwkang** [comment]
Oh.. It is my fault.
This request includes also [ZEPPELIN-754]Support seoul region.
Because I push '0d98c98' commit before Close this reqeust.
Is it okay?
Or What I have to do?

**Comment by Leemoonsoo** [comment]
@mwkang I think ZEPPELIN-754 is not much related to ZEPPELIN-751. Then how about remove last commit about ZEPPELIN-754 and create a new pullrequest for it?

**Comment by mwkang** [comment]
Okay. I remove last commit.

File: zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/S3NotebookRepo.java [hunk 1]
```diff
@@ -102,6 +102,16 @@ public S3NotebookRepo(ZeppelinConfiguration conf) throws IOException {
               if (info != null) {
                 infos.add(info);
               }
+            } catch (AmazonServiceException ase) {
+              LOG.info("Caught an AmazonServiceException for some reason.\n" +
```
Comment (felixcheung) [inline zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/S3NotebookRepo.java]: shouldn't these be warnings?

Comment (mwkang) [inline zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/S3NotebookRepo.java]: Um.. Warnings is better than info.
I will change.

**Comment by felixcheung** [comment]
looks good!

**Comment by Leemoonsoo** [comment]
Merging it into the master

**Pull request closed without merging**